livekit-agents/livekit/agents/cli/cli.py

```diff
@@ -234,6 +234,7 @@ async def _run_loop():
 
 def run_worker(args: proto.CliArgs) -> None:
     setup_logging(args.log_level, args.devmode)
+    args.validate_config()
```

Comment:
it'd be better to make the call directly vs another abstraction

```suggestion
    args.opts.validate_config()
```

Reply:
I've updated according to your advices